packages/@yoda/util/_.js

```diff
@@ -51,6 +51,17 @@ function startsWith (str, search, pos) {
   return str.substring(!pos || pos < 0 ? 0 : +pos, search.length) === search
 }
 
+module.exports.endsWith = endsWith
+function endsWith (str, search, length) {
+  if (typeof str !== 'string') {
+    return false
+  }
+  if (length === undefined || length > str.length) {
+    length = str.length
+  }
+  return str.substring(length - search.length, length) === search
+}
```

Comment:
This seems a function from es6, we might have a polyfill at YodaRT :)

Reply:
Test cases have been included and ran on CI. It's available on ShadowNode since very early version.

Reply:
I mean the function `String.prototype.endsWith`.

Reply:
Nope, doesn't work for me in my repl case :(